app/src/main/java/org/fossasia/susi/ai/login/LoginLogoutModulePresenter.kt

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.susi.ai.login
+
+import android.app.Activity
+import org.fossasia.susi.ai.data.UtilModel
+import org.fossasia.susi.ai.data.db.DatabaseRepository
+import org.fossasia.susi.ai.data.db.contract.IDatabaseRepository
+import org.fossasia.susi.ai.login.contract.ILoginLogoutModulePresenter
+import org.fossasia.susi.ai.skills.settings.contract.ISettingsView
+
+class LoginLogoutModulePresenter(activeActivity: Activity) : ILoginLogoutModulePresenter {
```

Comment:
Why do you need Activity, why not context?

Reply:
Sir,I have changed it. Please review now.